### ISSUE TITLE
Highlighted calendar items 

### ISSUE THREAD

**Issue by gsloan** [fluenttechnology/northstar#35]
Are these supposed to display more information when hovered?
![image](https://user-images.githubusercontent.com/5477755/45751819-6b8f0f80-bc0b-11e8-981e-f6084b6188c1.png)

**Comment by Chris-Shields** [comment]
Yes, they are meant to show a tooltip of the scheme / review it refers to

**Comment by Chris-Shields** [comment]
I've added some illustrations to the NorthStar Sharepoint

**Comment by goofballLogic** [comment]
@gsloan Note that the spec for this is changed from a tooltip to a modal dialog!
